firebase-firestore/src/main/java/com/google/cloud/datastore/core/number/NumberComparisonHelper.java

```diff
@@ -77,25 +59,14 @@ public static int compareLongs(long leftLong, long rightLong) {
     }
   }
 
-  /** Compares ints. See the comment to {@link #compareLongs} for rationale. */
-  public static int compareInts(int leftInt, int rightInt) {
```

Comment:
This is still in use so we can't remove it (though I guess we could also migrate the caller to calling compareLongs)

Reply:
Yes. this was a part of a refractor. Sorry for creating the pr too early.